src/components/map/Map.tsx

```diff
@@ -154,7 +154,7 @@ class LeafletMap extends React.Component<IMapProps, IMapState> {
                             `
                         }
                         baseLayer={true}
-                        maxZoom={23}
+                        maxZoom={19}
                         minZoom={6}
```

Comment:
Should we change this to 9 or 10 while we're at it?

Reply:
nah